src/FakeItEasy.Net35/Compatibility/ConcurrentQueue.cs

```diff
@@ -0,0 +1,34 @@
+namespace FakeItEasy

```

Comment:
I worked pretty hard on this in #568, and was unable to make it happen.
Although with your inspiration, I may have done. Don't merge, but like the latest commit?